lib/public/IDBConnection.php

```diff
@@ -45,6 +45,11 @@
  * @since 6.0.0
  */
 interface IDBConnection {
+	public const MYSQL = 'mysql';
```

Comment:
```suggestion
	public const PLATFORM_MYSQL = 'mysql';
```

so the constants can be used in typing

all constants need an `@since` tag